README.md

```diff
@@ -41,6 +59,10 @@ jupyter lab
 
 Note this library currently includes `rio.open` so must be run with an AWS identity that has access to the bucket the biomass products are in.
 
+## Links
+* [MAAP Visualization Discovery Report](https://docs.google.com/presentation/d/1Oz2Uh-jYEvPQlEgym8l6k45LvqqqdLwGy3atrZkLhJg/edit#slide=id.p1)
+
```

Comment:
❓ : Should this be included? I thought it would be helpful but i know this is a public repository, outsiders may not have access and if not, should this even be included or is this just for internal knowledge?

Reply:
This is likely meant to stay as internal knowledge documenting the evolving process of this project, unless @heidimok or @wildintellect have other thoughts on that. I can confirm that the presentation _is_ currently restricted from public view. This should be added to the onboarding doc though (if it's not already there.)

Reply:
I think the discovery report itself would just be an internal reference because it's more focused on MAAP's origins and also us just figuring out our scope of work. It can be helpful as our team needs to onboard or write blog posts etc about this project though.

Reply:
removing from README and confirming it is in the onboarding doc